bin/kano-ui-autostart

```diff
@@ -38,6 +38,13 @@ function track_startup
     # Report a startup event to Kano Tracker
     kano-tracker-ctl +1 startup
 
+    # If the kit comes with the Kano Screen Kit, track it
+    # FIXME: The query below needs testing on real hardware
+    screen_kit=`python -c "from kano_settings.system import display; print display.get_model()"`
+    if [screen_kit == "" ]; then
```

Comment:
Testing this condition on a flashed SK seems incorrect as it returns `HDMI1`. Judging from the `/boot/config.txt` we have two edid names which we go by when detecting SKs: `MST-HDMI1`, `MST-HDMI1`.

So the following command will give you the codes above.
```
python -c "from kano_settings.system import display; print display.get_edid_name(use_cached=False)"
``` 

However, I would make a function in `kano_settings.display` that would return whether an SK is plugged which you could then use above.